docbrown/core/src/tgraph_shard.rs

```diff
@@ -182,7 +183,7 @@ impl TGraphShard {
         let tgshard = self.rc.clone();
         let iter: GenBoxed<u64> = GenBoxed::new_boxed(|co| async move {
             let g = tgshard.read();
-            let iter = (*g).vertex_ids_window(w).map(|v| v.into());
+            let iter = (*g).vertex_ids_window(w).map(|v| v as u64);
```

Comment:
what type is `v` ?

Reply:
at this point v is a u64, into prevents it from compiling complaining about the type